docs/cow-protocol/tutorials/cow-swap/native.mdx

```diff
@@ -36,16 +40,32 @@ Wrapping the native token is an on-chain transaction and requires a gas fee.
 After wrapping the token, you can simply select the wrapped token as your sell token and continue with your swap.
 
 
-## Eth-flow
+:::tip
+
+If you plan to sell the native tokens over multiple orders, its best to wrap all it together, so you only pay the gas fee once.
+
+If this is a one-off swap, consider using Eth-flow instead.
+
+:::
+
+
+## Option 2) Use Eth-flow
 
 In an attempt to smooth the user experience, CoW Protocol has introduced the [Eth-flow](/cow-protocol/reference/contracts/periphery/eth-flow) contract. This allows users to automate the [above process](#wrap-native-tokens) and wrap + swap in a single **on-chain** transaction. 
 
-Using the Eth-flow is faster than wrapping, then swapping. However, because an Eth-flow transactions is on-chain (unlike most CoW Swap swaps), the user must pay an up-front gas fee in the native token. This fee is not refunded if the order fails.
+Eth-flow get the following benefits:
+1. Lower overall fees (if you need to swap just once)
+2. Simpler UX (single transaction)
+3. You don't need to approve WETH explicitly in order to sell it
 
-To use the Eth-flow, first select the native token you want to swap (e.g. `ETH`) and the token you want to buy (e.g. `COW`). Then click "Swap". CoW Protocol will automatically detect if the native token needs to be wrapped and will execute the Eth-flow contract on your behalf.
+Using the Eth-flow is faster than wrapping, then swapping. However, because an Eth-flow transactions is on-chain (unlike most CoW Swap swaps), the user must pay an up-front gas fee in the native token. This fee is not refunded if the order fails.
```

Comment:
```suggestion
Using the Eth-flow is faster than wrapping, then swapping. However, because an Eth-flow transactions is on-chain (unlike most CoW Swap swaps), you must pay an up-front gas fee in the native token. This fee is not refunded if the order fails.
```